contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -126,20 +88,24 @@ contract PBHVerifierImplV1 is WorldIDImpl {
     ///
     /// @param _worldId The World ID instance that will be used for verifying proofs. If set to the
     ///        0 addess, then it will be assumed that verification will take place off chain.
+    /// @param _entryPoint The ERC-4337 Entry Point.
     /// @param _numPbhPerMonth The number of allowed PBH transactions per month.
     ///
     /// @custom:reverts string If called more than once at the same initialisation number.
-    function initialize(IWorldIDGroups _worldId, uint8 _numPbhPerMonth) public reinitializer(1) {
+    function initialize(IWorldIDGroups _worldId, IEntryPoint _entryPoint, uint8 _numPbhPerMonth)
+        external
+        reinitializer(1)
```

Comment:
is the best practice to use `initializer` or `reinitializer(1)` for the first init?

Reply:
I think you use reinitializer to be able to upgrade the initialization steps itself